python/ray/train/session.py

```diff
@@ -231,6 +236,18 @@ def checkpoint(self, **kwargs):
         # checkpoint has been processed.
         self.continue_lock.acquire()
 
+    @property
+    def accelerator(self) -> Accelerator:
```

Comment:
The accelerator property is implemented with the following requirement in mind:
> To prevent undefined behavior, only one `Accelerator` instance can be used per training job.